bundle-size-chart/app.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright 2019, the AMP HTML authors
```

Comment:
Should this file be uploaded to GitHub?

Reply:
Yeah, there's nothing secret about this file. I dunno why I excluded it from the other apps...